main_windows.go

```diff
@@ -124,10 +127,35 @@ func runGraphicalUi() (err error) {
 
 	// Handle input
 	dlg.ui.rconInput.KeyPress().Attach(func(key walk.Key) {
+		// handle history (arrow up/down)
+		if key == walk.KeyUp || key == walk.KeyDown {
+			if len(history) == 0 {
+				return
+			}
+			
+			if key == walk.KeyUp {
+				if historyIndex == 0 { 
+					return
+				}
+				
+				historyIndex -= 1
+				dlg.ui.rconInput.SetText(history[historyIndex])
+			}else{
```

Comment:
Running `gofmt` on this file will insert consistent spacing for cases like this.